web/templates/settings/bot_avatar_row.hbs

```diff
@@ -39,14 +39,14 @@
             <div class="api-key-value-and-button no-select">
                 <!-- have the `.text-select` in `.no-select` so that the value doesn't have trailing whitespace. -->
                 <span class="value text-select">{{api_key}}</span>
-                <button type="submit" class="button no-style btn-secondary regenerate_bot_api_key" title="{{t 'Generate new API key' }}" data-user-id="{{user_id}}">
+                <button type="submit" class="button no-style btn-secondary regenerate_bot_api_key tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Generate new API key' }}" data-user-id="{{user_id}}">
                     <i class="fa fa-refresh" aria-hidden="true"></i>
                 </button>
             </div>
             <div class="api_key_error text-error"></div>
         </div>
         {{else}}
-        <button class="button round btn-warning reactivate_bot" title="{{t 'Reactivate bot' }}" data-user-id="{{user_id}}">{{t "Reactivate bot" }}</button>
+        <button class="button round btn-warning reactivate_bot" data-user-id="{{user_id}}">{{t "Reactivate bot" }}</button>
```

Comment:
This is now correct but the commit message should mention that this is not the same as the others.

Reply:
The new commit message is wrong -- the previous tooltip was an "HTML title tooltip" -- i.e. the built-in behavior of the `title` attribute -- it has nothing to do with bootstrap.

(I think an older issue writeup had this error too which may be how you got confused). https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title